keyboards/easyconvert/info.json

```diff
@@ -0,0 +1,140 @@
+{
+    "keyboard_name": "EASY-CONVERT",
+    "manufacturer": "LiGuo Studio",
+    "url": "",
+    "maintainer": "Cheng Leeren",
+    "usb": {
+        "vid": "0x1433",
```

Comment:
This VID is [already used](https://the-sz.com/products/usbid/index.php?v=0x1433). The suggestion is values over 0x4000.

Reply:
Thank you for your suggestion, but I seem unable to make any changes.

Reply:
Why not?  Technical limitation, or? 

For github, committing to the branch you used for the PR (in this case `master`) and pushing to that branch will update the PR.

Reply:
No, my firmware has already been released，now all the firmware burned in the keyboard is with a VID of 1433.

Reply:
The problem is that the vendor ID belongs to "TRANWO TECHNOLOGY CORP", per USB-IF. So they have own the right to use the ID, and only they do. 

In theory, each vendor should have a purchased vendor ID from USB-IF, but that's not realistic.  Especially for hobbyists and small vendors.  So, to avoid potential issues, we actively avoid vendor IDs that are actually in use unless the vendor has their own vendor ID.  That's why the suggested value is over 0x4000, since that's above what is being actively assigned. 

And I am guessing that the main issue here is with VIA/remap compatibility, since that matches based on VID/PID, rather than something less problematic.  Otherwise, there should be no issue with changing the VID. 

However, I personally do not feel comfortable accepting any configs that are hijacking vendor IDs.

Reply:
Okay, I will modify it later and upload it later

Reply:
I have changed the vid from 0x1433 to 0x4433